config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+            "fogflow/designer",
+            "fogflow/worker",
+            "fogflow/master",
+            "fogflow/broker",
+            "fogflow/discovery",
+            "fogflow/sum",
+            "fogflow/facecounter",
+            "fogflow/facefinder",
+            "fogflow/recommender",
+            "fogflow/connectedcar",
+            "fogflow/privatesite",
+            "fogflow/publicsite",
+            "fogflow/anomaly",
+            "fogflow/counter",
+            "fogflow/dummy",
+            "fogflow/nodejs"
+        ]
+    },
+    {
+        "enabler": "GIS Data Provider ",
+        "catalogue": "gis-data-provider-geoserver3d",
+        "academy": "GIS",
+        "readthedocs": "gisdataprovider",
+        "helpdesk": "GIS Provider",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Cyberlightning/geoserver",
+            "Cyberlightning/GISDataProvider"
+        ],
+        "docker": [
+            "fiware/gisdataprovider"
+        ]
+    },
+    {
+        "enabler": "IaaS GE",
+        "catalogue": "iaas-ge-fiware-reference-implementation",
+        "academy": "IaaS Resource Management",
+        "readthedocs": "",
+        "helpdesk": "IaaS RM",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [],
+        "docker": []
+    },
+    {
+        "enabler": "Idra",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "idra",
+        "helpdesk": "IDRA",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "OPSILab/Idra"
+        ],
+        "docker": [
+            "idraopendata/idra"
+        ]
+    },
+    {
+        "enabler": "Interface Designer",
+        "catalogue": "interface-designer",
+        "academy": "Interface Designer",
+        "readthedocs": "interface-designer",
+        "helpdesk": "UI Designer",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Adminotech/fiware-interface-designer"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "IoT Agent Node.JS Library",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "iotagent-node-lib",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/iotagent-node-lib"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "IoT Agent for JSON",
+        "catalogue": "",
+        "academy": "IDAS",
+        "readthedocs": "fiware-iotagent-json",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/iotagent-json"
+        ],
+        "docker": [
+            "fiware/iotagent-json"
+        ]
+    },
+    {
+        "enabler": "IoT Agent for Lightweight M2M",
```

Comment:
"enabler": "IoT Agent for LWM2M"

Reply:
Fixed in https://github.com/flopezag/fiware-tsc-dashboard/pull/38/commits/55ea9ac2cb5f6bb623a5b2897a9d1e5dede5c585